firestore-incremental-capture/functions/__tests__/functions.test.ts

```diff
@@ -0,0 +1,110 @@
+import * as admin from 'firebase-admin';
+import * as functions from 'firebase-functions-test';
+import {syncData} from '../src/index';
+import {getTable, initialize} from '../src/bigquery';
+
+import config from '../src/config';
+import {Table} from '@google-cloud/bigquery';
+import {clearBQTables} from './helpers';
+
+jest.mock('../src/config', () => ({
+  default: {
+    table: '',
+    dataset: '',
+    datasetLocation: 'us',
+    syncCollectionPath: 'testing',
+  },
+}));
+
+/** Setup project config */
+const projectId = 'dev-extensions-testing';
+const fft = functions({projectId});
+
+process.env.FIRESTORE_EMULATOR_HOST = '127.0.0.1:8080';
+process.env.FIREBASE_FIRESTORE_EMULATOR_ADDRESS = '127.0.0.1:8080';
+process.env.FIREBASE_AUTH_EMULATOR_HOST = '127.0.0.1:9099';
+process.env.PUBSUB_EMULATOR_HOST = '127.0.0.1:8085';
+process.env.GOOGLE_CLOUD_PROJECT = 'demo-test';
+process.env.FIREBASE_STORAGE_EMULATOR_HOST = '127.0.0.1:9199';
+
+/** Global vars */
+const {makeDocumentSnapshot} = fft.firestore;
+
+const db = admin.firestore();
+const collection = db.collection(config.syncCollectionPath);
+let randomId = '';
+
+xdescribe('functions', () => {
+  beforeAll(async () => {
+    /** clear all datasets */
+    await clearBQTables();
+  });
+  beforeEach(async () => {
+    /** generate random id */
+    randomId = (Math.random() + 1).toString(36).substring(7);
+
+    config.table = randomId;
+    config.dataset = randomId;
+
+    await initialize();
+  });
+
+  xit('Can sync data with BQ', async () => {
```

Comment:
```suggestion
  xit('Can sync data with BQ', async () => {
```

These tests may now be obsolete and can be removed.